docs/D-Modularity/functions.md

```diff
@@ -540,29 +552,29 @@ int getNonNegInt(void)
 //
 int power(int base, int exponent)
 {
-        int result, i;
+    int result, i;
 
-        result = 1;
-        for (i = 0; i < exponent; i++)
-                result = result * base;
+    result = 1;
+    for (i = 0; i < exponent; i++)
+            result = result * base;
 
-        return result;
+    return result;
 }
 
 int main(void)
 {
-        int base, exp, answer;
+    int base, exp, answer;
 
-        printf("Enter base : ");
-        scanf("%d", &base);
+    printf("Enter base : ");
+    scanf("%d", &base);
 
-        printf("Enter exponent\n");
-        exp = getNonNegInt();
-        answer = power(base, exp);
+    printf("Enter exponent\n");
+    exp = getNonNegInt();
+    answer = power(base, exp);
         
-        printf("%d^%d is %d\n", base, exp, answer);
+    printf("%d^%d is %d\n", base, exp, answer);
 
-        return 0;
+    return 0;
```

Comment:
I think this has an output based on https://ict.senecacollege.ca/~ipc144/pages/content/modul.html. Maybe we should add that too

Reply:
Hmmmm do you have any suggestions about how to do this? On the IPC website they use tables for everything and in the HTML it's set as "width 95%" and the code output picture is actually a `<pre>` element inside a `<td>`... Do you have any tips for how to get this output on the page?

@humphd One thing I could do but it involves changing the page layout is just add an image of the output underneath, instead of beside. If there's a .png or .jpg source file somewhere. Unless you or Kevan can think of a way to make this markdown image half width to get rid of the blank space and then put the output img next to it? I'm kind of stumped on this one.

Reply:
I think you can put the output below, instead of having on the side

Reply:
I think I can do that if the professor is cool with that. I just don't know if there's an image file for just that part, or if I should just take a screenshot of it... or I could literally just type out the output in Markdown. One of those 2 things.

Reply:
Hey @Kevan-Y I think I made it look pretty good. I used level 3 headings similar to the other parts of the page. I also got rid of the yellow table inline styles so now it looks good on dark and light mode.

Let me know if it looks good :)

Reply:
Looks good to me.